functional_tests/scylla_operator/pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
 markers =
-    require_node_terminate: test require certain node terminate method, if backend does not support it, test will be skipped
-    require_mgmt: test require scylla manager to be provisioned, set "use_mgmt: true" to enable it
+    requires_node_termination_support: test gets skipped if backend doesn't support specified K8S node termination method
```

Comment:
It'll be nice to define it with parameter:

"requires_node_termination_support(terminate_method, ...): test gets skipped if backend doesn't support specified K8S node termination method"

wdyt?

Reply:
It's up to you.